src/submission-service.ts

```diff
@@ -346,10 +347,19 @@ async function executeCancelAction(
     isUploadingAttachments: false,
   }
 
+  let postSubmissionAction =
+    formSubmissionResult.definition.postSubmissionAction
```

Comment:
This should be `cancelAction`

Reply:
Another classic Josh copy and paste today